packages/mux-player/package.json

```diff
@@ -15,18 +23,19 @@
   "scripts": {
     "clean": "shx rm -rf dist/",
     "lint": "eslint src/ --ext .js,.jsx,.ts,.tsx",
-    "test": "web-test-runner **/*test.js --port 8001 --coverage --config test/web-test-runner.config.mjs",
+    "test": "web-test-runner **/*test.js --port 8001 --coverage --config test/web-test-runner.config.mjs --root-dir ../..",
     "posttest": "replace 'SF:src/' 'SF:packages/mux-player/src/' coverage/lcov.info --silent",
     "i18n": "yarn build:esm --keep-names && i18n-utils dist/index.mjs ./lang",
     "dev:iife": "open-process | yarn build:iife --watch",
     "dev:esm": "open-process | yarn build:esm --watch",
     "dev:types": "yarn build:types -w",
     "dev": "npm-run-all --parallel dev:types dev:esm dev:iife",
     "build:esm": "esbuilder src/index.ts --format=esm --out-extension:.js=.mjs",
+    "build:esm-module": "esbuilder src/index.ts --format=esm-module --out-extension:.js=.mjs",
     "build:iife": "esbuilder src/index.ts --format=iife --out-extension:.js=.js",
     "build:cjs": "esbuilder src/index.ts --format=cjs --out-extension:.js=.cjs",
     "build:types": "tsc --declaration --emitDeclarationOnly --outDir './dist/types'",
-    "build": "npm-run-all --parallel build:types 'build:esm -- --minify' 'build:iife -- --minify' 'build:cjs -- --minify'",
+    "build": "npm-run-all --parallel build:types 'build:esm -- --minify' 'build:iife -- --minify' 'build:cjs -- --minify' 'build:esm-module -- --minify'",
```

Comment:
missing metafiles for all of these

Reply:
Handled in esbuilder https://github.com/muxinc/elements/pull/163/files#diff-13a7e6d07a00292b32c2c4dbc20051312abb834f12127b82ec7069ad613e51cbR89-R90